subgraphs/_reference_/src/sdk/protocols/bridge/tokens.ts

```diff
@@ -10,7 +10,7 @@ import { Bridge } from "./protocol";
 import { RewardTokenType } from "../../util/constants";
 
 export interface TokenInitializer {
-  getTokenParams(address: Address): TokenParams;
+  getTokenParams(address: Bytes): TokenParams;
```

Comment:
We shouldn't be changing the interfaces that subgraphs need to implement. Because if now we wanted to upgrade the SDK version a given subgraph is using it will break its implementation

Reply:
I suppose in this case it is better to use the more general `Bytes`? Since Address inherits from Bytes, Address type should still work.

Reply:
I'd have to test it, but can you do `class Class implements Interface` on a class in which one of its methods doesn't exactly match the signature of the interface? Even if the difference lays in a type that inherits from another? :P 

Does `getTokenParams(address: Address): TokenParams` implement `getTokenParams(address: Bytes): TokenParams`?

I have the opposite opinion here 😅 , I think it is better to use "less general" constraints whenever possible. It removes ambiguity and potential source of errors. If we wanted to be as generic as possible we wouldn't be using types. If there are no scenarios in which we need to pass Bytes, I don't think we should support it

Reply:
My test is that I can define the interface using Bytes type, but implements it using Address type. And it doesn't work the other way around.

Reply:
But I don't think I need the tokenParams to be Bytes anyway (see https://github.com/messari/subgraphs/pull/1768#discussion_r1112519056), I will revert this change.